includes/class-webmention-receiver.php

```diff
@@ -205,9 +215,9 @@ public static function default_request_handler( $data ) {
 		status_header( apply_filters( 'webmention_success_header', 200 ) );
```

Comment:
With the standard stable, do we need this filter? Any scenario that wouldn't return a 200 on success would require a different request handler.